sklearn/model_selection/partition.py

```diff
@@ -0,0 +1,1129 @@
+"""
+The :mod:`sklearn.model_selection.partition` module includes classes and
```

Comment:
On similar lines do you think I should rather revert to the previously accepted name `validate` rather than `validation` (which was my change)? 

i.e (`spit`, `search`, `validate`) instead of (`split`, `search`, `validation`)?